fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchTokenTest.java

```diff
@@ -328,6 +368,30 @@ public void testSearchToken_Coding_chained_missing() throws Exception {
         assertSearchDoesntReturnComposition("subject:Basic.missing-Coding:missing", "false");
     }
 
+    @Test
+    public void testSearchToken_Coding_in() throws Exception {
+        assertSearchDoesntReturnSavedResource("Coding:in", "http://hl7.org/fhir/ValueSet/concept-property-type");
```

Comment:
For these Code/Coding/CodeableConcept tests. Maybe it would be good to have at least one ":in" test that does return the saved resource and at least one ":not-in" test that doesn't return the saved resource, to cover both sides.